package.json

```diff
@@ -27,18 +30,20 @@
   },
   "homepage": "https://github.com/ciena-frost/frost-standard#readme",
   "devDependencies": {
-    "chai": "^3.4.1",
-    "eslint": "^3.0.1",
+    "chai": "^4.0.0",
+    "eslint": "^4.1.0",
     "istanbul": "^0.4.2",
-    "mocha": "^2.4.2"
+    "mocha": "^3.4.2"
   },
   "dependencies": {
-    "babel-eslint": "^6.1.2",
-    "eslint-config-standard": "^5.3.5",
+    "babel-eslint": "^7.2.3",
+    "eslint-config-standard": "^10.2.1",
```

Comment:
Well, I went from a major of `5` to a major of `10` I would hope they don't introduce lots of new rules in minors, we haven't seen that yet at least. But I have no problem pinning it.